src/algebra/lie/basic.lean

```diff
@@ -891,6 +926,84 @@ end lie_span
 
 end lattice_structure
 
+section lie_ideal_operations
+
+/-- Given a Lie module `M` over a Lie algebra `L`, the set of Lie ideals of `L` acts on the set
+of submodules of `M`. -/
+instance : has_bracket (lie_ideal R L) (lie_submodule R L M) :=
+⟨λ I N, lie_submodule.lie_span R L { m | ∃ (x : I) (n : N), ⁅(x : L), (n : M)⁆ = m }⟩
+
+lemma lie_apply :
```

Comment:
why exactly this name?

Reply:
A moment of madness? This is obviously a terrible name. I have just pushed a commit renaming it to `lie_ideal_oper_eq_span` which seems better but still not ideal.